webapp/src/ts/app.component.ts

```diff
@@ -278,7 +279,12 @@ export class AppComponent implements OnInit, AfterViewInit {
       .then(() => this.initForms())
       .then(() => this.initUnreadCount())
       .then(() => this.checkDateService.check(true))
-      .then(() => this.startRecurringProcesses());
+      .then(() => this.startRecurringProcesses())
+      .catch(err => {
+        console.error('Error during initialisation', err);
+        this.showContent = true;
+        this.router.navigate(['/error', '418' ]);
```

Comment:
I'm a teapot???

I feel like a 503 "Service Unavailable" might be more appropriate  :P